### PR TITLE
update unity griffon docs

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**

### CODE REVIEW

File: beta/project-griffon/set-up-project-griffon/README.md [hunk 1]
```diff
@@ -254,5 +285,27 @@ ACPGriffon.startSession(sessionUrl, function(handleCallback) {
...
+{% tab title="Unity" %}
+
+### startSession
+
+#### C#
+
+#### Syntax
+
+```csharp
+ACPGriffon.StartSession(url);
```
Comment (rymorale) [inline beta/project-griffon/set-up-project-griffon/README.md:298]: This syntax should match the format seen at https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/308/files#diff-3cb62446150e11a29b0f96634b637df3R54

File: beta/project-griffon/set-up-project-griffon/project-griffon-sdk-api-reference.md [hunk 2]
```diff
@@ -100,6 +100,28 @@ ACPGriffon.startSession(url, function(handleCallback) {
...
+{% tab title="Unity" %}
+
+### startSession
+
+#### C#
+
+#### Syntax
+
+```csharp
+ACPGriffon.StartSession(url);
```
Comment (rymorale) [inline beta/project-griffon/set-up-project-griffon/project-griffon-sdk-api-reference.md:113]: This syntax should match the format seen at https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/308/files#diff-3cb62446150e11a29b0f96634b637df3R54

File: beta/project-griffon/set-up-project-griffon/README.md (hunk 1 above)
Comment (spoorthipujariadobe) [reply beta/project-griffon/set-up-project-griffon/README.md]: done

File: beta/project-griffon/set-up-project-griffon/project-griffon-sdk-api-reference.md (hunk 2 above)
Comment (spoorthipujariadobe) [reply beta/project-griffon/set-up-project-griffon/project-griffon-sdk-api-reference.md]: done

**Pull request merged**